NtupleProducer/plugins/HTauTauNtuplizer.cc

```diff
@@ -2688,13 +2714,13 @@ int HTauTauNtuplizer::FillJet(const edm::View<pat::Jet> *jets, const edm::Event&
     // 2017 and 2018 data
     else if (theYear == 2017 || theYear == 2018)
     {
-      if (absjeta <= 2.4)
+      if (absjeta <= 2.6)
```

Comment:
Can you please clarify why you are changing the value, maybe providing a reference? It just follows from a preexisting binning? 
FYI, in the [skimming](https://github.com/LLRCMS/KLUBAnalysis/blob/73763a47900cecbfd726fb78a4b0c5e6c73441e3/test/skimNtuple_HHbtag.cpp#L4372) we are using 2.5 as threshold.

Reply:
The reference for the cuts on the jet ID [1] is (and was) already there above the eta if blocks. 

[1] https://twiki.cern.ch/twiki/bin/view/CMS/JetID13TeVUL